### ISSUE TITLE
Muon analysis failing with numba error

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/cta-lstchain#854]
```
            )
E           ValueError: Running lstchain_data_r0_to_dl1 failed with return code 1, output: 
E            Found duplicated column obs_id, skipping
E           Found duplicated column event_id, skipping
E           Traceback (most recent call last):
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/bin/lstchain_data_r0_to_dl1", line 33, in <module>
E               sys.exit(load_entry_point('lstchain', 'console_scripts', 'lstchain_data_r0_to_dl1')())
E             File "/home/maxnoe/Uni/CTA/cta-lstchain/lstchain/scripts/lstchain_data_r0_to_dl1.py", line 189, in main
E               r0_to_dl1.r0_to_dl1(
E             File "/home/maxnoe/Uni/CTA/cta-lstchain/lstchain/reco/r0_to_dl1.py", line 581, in r0_to_dl1
E               analyze_muon_event(subarray,
E             File "/home/maxnoe/Uni/CTA/cta-lstchain/lstchain/image/muon/muon_analysis.py", line 217, in analyze_muon_event
E               intensity_fitter(1,
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/lib/python3.8/site-packages/ctapipe/image/muon/intensity_fitter.py", line 553, in __call__
E               minuit.migrad()
E             File "src/iminuit/_libiminuit.pyx", line 886, in iminuit._libiminuit.Minuit.migrad
E           RuntimeError: exception was raised in user function
E           User function arguments:
E                      impact_parameter = +5.547522
E                                   phi = +0.000000
E                              center_x = +0.001723
E                              center_y = +0.007285
E                                radius = +0.018902
E                            ring_width = +0.002813
E               optical_efficiency_muon = +0.100000
E           Original python exception in user function:
E           UFuncTypeError: (<ufunc 'chord_length'>, (<class 'numpy.dtype[float64]'>, <class 'numpy.dtype[float64]'>, <class 'numpy.dtype[float64]'>, <class 'numpy.dtype[float64]'>))
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/lib/python3.8/site-packages/ctapipe/image/muon/intensity_fitter.py", line 396, in negative_log_likelihood
E               prediction = image_prediction_no_units(
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/lib/python3.8/site-packages/ctapipe/image/muon/intensity_fitter.py", line 258, in image_prediction_no_units
E               ang_prof, profile = create_profile(
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/lib/python3.8/site-packages/ctapipe/image/muon/intensity_fitter.py", line 141, in create_profile
E               length = intersect_circle(mirror_radius, impact_parameter, ang, hole_radius)
E             File "/home/maxnoe/.local/anaconda/envs/lst-dev/lib/python3.8/site-packages/ctapipe/image/muon/intensity_fitter.py", line 87, in intersect_circle
E               mirror_length = chord_length(mirror_radius, (r / mirror_radius), angle)
```

Setting up a fresh env with the packages we need from conda-forge fails with this numba error, I was not yet able to pin down.
Reproduced locally and happening here: #852

**Comment by maxnoe** [comment]
Seems to be an issue in numpy 1.22: https://github.com/numpy/numpy/issues/20735

**Comment by rlopezcoto** [comment]
solved by fixing numpy version to <1.22 in #852
